### ISSUE TITLE
Aternos Server Not Monitoring Well

### ISSUE THREAD

**Issue by jezto** [RahulR100/mcstatusbot#55]
Aternos server status is not showing correctly. The server is started and there is a player in the server. But the bot shows that the server is offline and there is no players. Proof included.
![Screenshot 2024-03-10 101757](https://github.com/RahulR100/mcstatusbot/assets/101259521/1efaed67-5b8c-40fb-bf58-d091a70209f0)
![Screenshot 2024-03-10 101910](https://github.com/RahulR100/mcstatusbot/assets/101259521/94a8177a-5c70-4883-94d2-b83744d0a469)

**Comment by RahulR100** [comment]
Yes the voice channels can take about 6 minutes to update. This is a discord limitation (you may use the `/status` command to get a more updated status. 

Aternos server are also know to be slightly unreliable due to strategies used by aternos (they should work 90% of the time but sometimes may be unreliable). We are looking for a fix but this is a tricky situation so a fix will take some time.

**Comment by GamerMP523** [comment]
i have the same problem

**Comment by RahulR100** [comment]
Most hosting providers seem to be working normally but as the bot grows we are seeing more problematic ones where the server is shown as online when not and vice-versa. We are looking into seeing if we can update our current system to work around this issue and we are most likely going to go with hosting specific settings (especially since aternos seems so popular now). Will post back here with more updates.
